incuna-sass/functions/_rem.sass

```diff
@@ -5,6 +5,8 @@ $use-rem: false !default
     @if $use-rem
         @if $target == 0
             @return 0
+        @if $target == 1px
+            @return 1px
```

Comment:
We haven't seen any problems using `2px` recently. Lets add that in later if it becomes an issue.